packages/suite/src/middlewares/suite/suiteMiddleware.ts

```diff
@@ -93,7 +93,9 @@ export const prepareSuiteMiddleware = createMiddlewareWithExtraDeps(
 
             dispatch(handleDeviceDisconnect(device));
             if (isTrezorDeviceWithState(device)) {
-                extra.services.suiteSync.unsubscribeSuiteSyncStorage({ device });
+                extra.services.suiteSync.turnOffSuiteSyncForWallet({
+                    owner: device.suiteSyncOwner,
+                });
```

Comment:
Duplication of what's in https://github.com/trezor/trezor-suite/blob/595cf7abd958437f0f9fba8a5ad284afc745a537/suite-native/device/src/middlewares/deviceMiddleware.ts#L54

Please unify this. Either in common device middleware or suite sync middleware 🙏